artifact.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PUBKY="https://api.bintray.com/orgs/jfrog/keys/gpg/public.key"
+REPOURL="https://jfrog.bintray.com/artifactory-debs"
+
+ echo "Installing Java for Jfrog Artifactory"
```

Comment:
Using coloured echo commands could greatly help at debugging the flow of the script.